src/matrix/verification/SAS/channel/types.ts

```diff
@@ -0,0 +1,25 @@
+export const enum VerificationEventTypes {
+    Request = "m.key.verification.request",
+    Ready = "m.key.verification.ready",
+    Start = "m.key.verification.start",
+    Accept = "m.key.verification.accept",
+    Key = "m.key.verification.key",
+    Cancel = "m.key.verification.cancel",
+    Mac = "m.key.verification.mac",
+    Done = "m.key.verification.done",
+}
+
+export const enum CancelTypes {
```

Comment:
Given the field where this is used is called `reason`, this might be a better name?
```suggestion
export const enum CancelReason {
```